azuread/resource_application.go

```diff
@@ -178,6 +177,56 @@ func resourceApplication() *schema.Resource {
 				},
 			},
 
+			"app_role": {
```

Comment:
Could we move this property to under type?

Reply:
sorted